### ISSUE TITLE
Run Tests Against Python 3.10

### ISSUE THREAD

**Issue by grant** [GoogleCloudPlatform/appengine-python-standard#36]
## Expected Behavior

Unit tests are run against python310. This would be nice to have as we're developing a runtime that uses

## Actual Behavior

Unit tests don't run against 3.10

https://github.com/GoogleCloudPlatform/appengine-python-standard/blob/6ea19dde91c149398c0d9aa9b8f5c1e6e65dfd9b/.github/workflows/unit.yml#L16

CC: @shreejad @asriniva

**Comment by asriniva** [comment]
Turns out ruamel.yaml doesn't support Python 3.10 yet, so we're waiting on that.
